### PR TITLE
[CLEANUP] Clean up `project.bower-dependencies` deprecation

### PR DESCRIPTION
**Pull request by bertdeblock**

### CODE REVIEW

File: lib/models/project.js [hunk 1]
```diff
@@ -404,31 +404,10 @@ class Project {
...
-          enabled: '4.3.0',
-        },
-        until: '5.0.0',
-      }
-    );
-
-    return this._bowerDependencies(bower);
-  }
-
   _bowerDependencies(bower) {
```
Comment (bertdeblock) [inline lib/models/project.js:411]: Will be cleaned up in a separate PR.

**Review by kategengler — APPROVED** [review]
